packages/variant/src/index.js

```diff
@@ -19,6 +19,7 @@ export const variant = ({
 
 export default variant
 
-export const buttonStyle = variant({ key: 'buttons' })
+export const buttonStyle =
+  variant({ key: 'buttons' }) || variant({ key: 'buttonStyles' })
```

Comment:
Sorry for the delay on this! It looks like Circle CI isn't running tests on some of the PRs, but this doesn't look like it would work 🤔 I might be missing something, but are the tests all passing locally?

Reply:
Yep, they are passing on local. I just set up tests on CI as well: https://circleci.com/gh/siddharthkp/styled-system/2

> but this doesn't look like it would work

Tell me more

Reply:
- `variant(args)` returns a function
- `funcA || funcB` will return `funcA`
- `buttonStyle` should always return the first function (unless I'm missing something, that is)

But just noticed your tests aren't testing this function here...

Reply:
> variant(args) returns a function

damn, got it

Reply:
Haha! For a second, I thought you were using something that I didn't know was possible in JS :)

Reply:
You still interested in working on this? If not, should we go ahead and close this out?